.github/workflows/release.yml

```diff
@@ -42,6 +42,7 @@ jobs:
           - metad
           - storaged
           - tools
+          - standalone
```

Comment:
should also change workflow in rc.yml.

workflow in `release` is just copy rc to release.